admin/config.yml

```diff
@@ -0,0 +1,23 @@
+backend:
+  name: github
+  repo: jduss4/elizabethzaleski
+  branch: main
+  site_domain: https://mysite.com
+  base_url: https://mysite.com
+  auth_endpoint: /api/auth
+media_folder: assets/cms
+public_folder: assets/cms/
```

Comment:
If Elizabeth uploads files via the CMS, they will land in the media_folder. Keeping these the same makes it easier to actually use the files without transformations. You might want to install Eleventy-Image as I [describe here](https://cassey.dev/adding-decap-cms-to-11ty/#next-steps) to make sure those images get nice and small if image upload is gonna be part of the final site.